server/libs/userlibs.js

```diff
@@ -48,4 +50,64 @@ exports.login = function(req,res){
     }).catch((error) => {
         res.status(error.status).send(error.message);
     });
+};
+
+exports.updateUser = function(req,res) {
+    let body = req.body;
+
+    const validationForNameExistence = (object) => {
+        if("name" in object){
+            return true;
+        }else{//eslint-disable-line
+            return false;
+        }
+    }
+
+    const validationForEmailExistence = (object) => {
+        if("email" in object){
+            return true;
+        }else{//eslint-disable-line
```

Comment:
same and apply on all the undiscussed disability

Reply:
Done.